src/fonduer/features/feature_libs/textual_features.py

```diff
@@ -69,46 +69,41 @@ def extract_textual_features(
             for f in _get_word_feats(span):
                 yield candidate.id, f"BASIC_{f}", DEF_VALUE
 
-        # Binary candidates
-        elif len(args) == 2:
-            span1, span2 = args
-            if span1.sentence.is_lingual() and span2.sentence.is_lingual():
-                get_tdl_feats = compile_relation_feature_generator()
-                sent1 = get_as_dict(span1.sentence)
-                sent2 = get_as_dict(span2.sentence)
-                xmltree = corenlp_to_xmltree(span1.sentence)
-                s1_idxs = list(
-                    range(span1.get_word_start_index(), span1.get_word_end_index() + 1)
-                )
-                s2_idxs = list(
-                    range(span2.get_word_start_index(), span2.get_word_end_index() + 1)
-                )
-                if len(s1_idxs) > 0 and len(s2_idxs) > 0:
+        # Multinary candidates
+        else:
+            spans = args
+            if all([span.sentence.is_lingual() for span in spans]):
+                get_tdl_feats = compile_relation_feature_generator(is_multary=True)
```

Comment:
For consistency, should this be `is_multinary`?

Reply:
This would require a change in the `treedlib` library and another release there, so I would say let's keep it like this.
Both `multary` and `multinary` are correct words. But when I first started developing this feature I used the term `multary`, and the new version of `treedlib` was released with that term.

You can find it here:
https://github.com/HazyResearch/treedlib/blob/master/treedlib/features.py#L4

Reply:
Sounds good to me :)